README.md

```diff
@@ -28,241 +25,141 @@ Currently, the tool supports the following project types:
-
-### Optional Features
-
-Some features within Upgrade Assistant may be enabled by adding extensions. Available extensions can be viewed on [here](https://www.nuget.org/packages?packagetype=UpgradeAssistantExtension&sortby=relevance&q=&prerel=True).
-
-**Loose Assembly Identification**
-
-Some projects incorporate what we call "loose assemblies" where assemblies are added directly to a repo. An experimental feature described [here](docs/design/Loose_binary_identification.md) can be used to help identify these and convert them to available NuGet packages.
-
-## Upgrade documentation
-
-As you upgrade projects from .NET Framework to .NET (STS, LTS, or preview), it will be very useful to be familiar with relevant [porting documentation](https://docs.microsoft.com/dotnet/core/porting/).
-
-Web scenarios can be especially challenging, so it you are upgrading an ASP.NET app, be sure to read [ASP.NET Core migration documentation](https://docs.microsoft.com/aspnet/core/migration/proper-to-2x). If you are unfamiliar with ASP.NET Core, you should also read [ASP.NET Core fundamentals documentation](https://docs.microsoft.com/aspnet/core/fundamentals) to learn about important ASP.NET Core concepts (hosting, middleware, routing, etc.).
-
-The following tutorials will give you a sense of how to upgrade ASP.NET, Windows Forms, and WPF applications to .NET (STS, LTS, or preview) using the Upgrade Assistant:
-- [Upgrade an ASP.NET MVC App to .NET 6 with the .NET Upgrade Assistant](https://docs.microsoft.com/en-us/dotnet/core/porting/upgrade-assistant-aspnetmvc)
-- [Upgrade a Windows Forms App to .NET 6 with the .NET Upgrade Assistant](https://docs.microsoft.com/en-us/dotnet/core/porting/upgrade-assistant-winforms-framework)
-- [Upgrade a WPF App to .NET 6 with the .NET Upgrade Assistant](https://docs.microsoft.com/en-us/dotnet/core/porting/upgrade-assistant-wpf-framework)
-- [Upgrade a UWP App to Windows App SDK with the .NET Upgrade Assistant](https://docs.microsoft.com/en-us/dotnet/core/porting/upgrade-assistant-uwp-framework) (This is still in preview. Read more [here](docs/uwp_support.md))
-
-Learn more about upcoming support for Xamarin.Forms to .NET MAUI upgrades [here](docs/maui_support.md). 
-
-Download this free e-book on [Porting existing ASP.NET apps to .NET Core](https://aka.ms/aspnet-porting-ebook).
-
-## Installation
-
-### Prerequisites
-
-1. This tool uses MSBuild to work with project files. Make sure that a recent version of MSBuild is installed. An easy way to do this is to [install Visual Studio](https://visualstudio.microsoft.com/downloads/).
-2. This tool requires that your project builds. This may include [installing Visual Studio](https://visualstudio.microsoft.com/downloads/) to ensure build SDKs (such as for web applications, etc) are available.
-
-### Installation steps
-
-The tool can be installed [from NuGet](https://www.nuget.org/packages/upgrade-assistant/) as a .NET CLI tool by running:
+### Extensibility
```

Comment:
>Extensibility [](http://example.com/codeflow?start=4&length=13)

as i mentioned above lets create a mapping specific section and describe how mappings specific work should be happening